x/blob/README.md

```diff
@@ -144,10 +144,10 @@ each PFB, to be included in a block must follow a set of validity rules.
    must be valid. This validity is determined by the following sub-rules:
     1. The namepsace of each blob must match the respective (same index)
        namespace in the `MsgPayForBlobs` `sdk.Msg` field `namespaces`.
-    1. The namespace is not lexicographically less than the [MAX_RESERVED_NAMESPACE](../../specs/src/specs/consensus.md#constants) range.
+    1. The namespace is lexicographically greater than the [MAX_RESERVED_NAMESPACE](../../specs/src/specs/consensus.md#constants).
     1. The namespace is not the
        [TAIL_PADDING_NAMESPACE](../../specs/src/specs/consensus.md#constants)
-       or [RESERVED_PADDING_NAMESPACE](../../specs/src/specs/consensus.md#constants).
+       or [PARITY_SHARE_NAMESPACE](../../specs/src/specs/consensus.md#constants).
```

Comment:
are these both not true?

Reply:
Yes they're both true.

https://github.com/celestiaorg/celestia-app/blob/a435dca97bab00593b54b90e7c74b8b6c2595ba3/pkg/namespace/consts.go#L49-L56

The MAX_RESERVED_NAMESPACE and RESERVED_PADDING_NAMESPACE are identical. So the statement above already states that a blob namespace must be greater than MAX_RESERVED_NAMESPACE (or RESERVED_PADDING_NAMESPACE).

Prior to this PR, the doc didn't clarify that the PARITY_SHARE_NAMESPACE can't be used by a blob namespace.

Reply:
ahh I see thanks for the clarification